fs/move.py

```diff
@@ -99,10 +132,10 @@ def move_dir(
     """
 
     def src():
-        return manage_fs(src_fs, writeable=False)
+        return manage_fs(src_fs, writeable=True)
 
     def dst():
-        return manage_fs(dst_fs, create=True)
+        return manage_fs(dst_fs, writeable=True, create=True)
 
     with src() as _src_fs, dst() as _dst_fs:
```

Comment:
Perhaps the `src()` and `dst` functions here are simple enough that the functions ought to be removed, and just move the code inline; like is done in `move_file` ?